### ISSUE TITLE
CA1003 reports non-generic events in Microsoft's code

### ISSUE THREAD

**Issue by pakdev** [dotnet/roslyn-analyzers#1087]
#### Analyzer package

Microsoft.ApiDesignGuidelines.Analyzers

#### Analyzer

UseGenericEventHandlerInstances

#### Repro steps

```cs
using System.ComponentModel;

public class MyViewModel : INotifyPropertyChanged
{
    public event PropertyChangedEventHandler PropertyChanged;

    protected virtual void OnPropertyChanged([CallerMemberName] string propertyName = null)
    {
        PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
    }
}
```

#### Expected behavior

Should not fire because PropertyChangedEventHandler is defined by Microsoft. Until Microsoft follows this rule themselves, there will be a bunch of false positives

#### Actual behavior

Fires

**Comment by alejandro5042** [comment]
I have a proposed fix to this issue in: https://github.com/dotnet/roslyn-analyzers/pull/1131

**Comment by mavasani** [comment]
Fix was merged. Thanks @alejandro5042!
